go/pkg/bertyprotocol/testing_test.go

```diff
@@ -37,3 +20,36 @@ func testSameErrcodes(t *testing.T, expected, got error) {
 		"%v", got,
 	)
 }
+
+func TestClient_impl(t *testing.T) {
+	var _ Service = (*service)(nil)
+	var _ ProtocolServiceServer = (*service)(nil)
+}
+
+func TestEmptyArgs(t *testing.T) {
+	// initialize new client
+	client, err := New(Opts{})
+	require.NoError(t, err)
+	err = client.Close()
+	require.NoError(t, err)
+}
+
+func TestTestingProtocol(t *testing.T) {
+	ctx := context.Background()
+	opts := TestingOpts{}
+	tp, cleanup := NewTestingProtocol(ctx, t, &opts)
+	assert.NotNil(t, tp)
+	cleanup()
+}
```

Comment:
It would be nice to test for goroutines leaks now that we need to support restarting the node services (maybe it's already the case)

Reply:
I started a PR in go-orbit-db -> https://github.com/berty/go-orbit-db/pull/42